layers/+tags/gtags/funcs.el

```diff
@@ -90,8 +90,8 @@ Otherwise does nothing."
     "gG" 'helm-gtags-dwim-other-window
     "gi" 'helm-gtags-tags-in-this-function
     "gl" 'helm-gtags-parse-file
-    "gn" 'helm-gtags-next-history
-    "gp" 'helm-gtags-previous-history
+    "gn" 'helm-gtags-go-forward
```

Comment:
It seems you have changed the wrong lines. It should be `counsel-gtags-go-forward` and not `helm-gtags-go-forward`. `helm-gtags` is well and fine. You need to change the keybindings for counsel-gtags.